lesson-7/challenge/index.css

```diff
@@ -0,0 +1,95 @@
+html{
+    height:100%;
+}
+body{
+    height:150%;
+    width:100%;
+    display:flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background: rgb(223, 237, 240);
+}
+.title--normal{
+    font-size:30px;
+    color:rgb(143, 142, 142);
+    word-spacing:130%;
+    margin-top:4%;
+}
+.title--bold{
+    font-size:31px;
+    color:black;
+    font-weight: bolder;
+    word-spacing: 130%;
+    margin:1%;
+}
+.subtext{
+    font-size:17px;
+    color:rgb(156, 156, 156);
+    width:35%;
+    text-align: center;
+
+}
+.container-grid__elem{
+    display:flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    background: rgb(248, 248, 248);
+    padding:5%;
+    border-radius: 1%;
+}
+.container-grid__elem p:nth-child(1){
```

Comment:
In this case, It's ok. But maybe prevent selecting the 'p' tag. You can select it by class name also.